ci/functional/job_cleanup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Common cleanup for running after a functional test
+
+set -ex
+
+rm -rf install/lib/daos/TESTING/ftest/avocado/job-results/job-*/html/
+
+# Remove the latest avocado symlink directory to avoid inclusion in the
+# jenkins build artifacts
+unlink install/lib/daos/TESTING/ftest/avocado/job-results/latest
+rm -rf "Functional/"
+mkdir "Functional/"
+# compress those potentially huge DAOS logs
+if daos_logs=$(find \
+    install/lib/daos/TESTING/ftest/avocado/job-results/job-*/daos_logs/* \
+    -maxdepth 0 -type f -size +1M); then
+  if  [ -n "daos_logs" ]; then
+    # shellcheck disable=SC2086
+    lbzip2 $daos_logs
```

Comment:
Wouldn't xargs work here?

Reply:
I can not find a way for it to work.  My tests with xargs show that it removes the quotes from the strings, which makes it equivalent to what shellcheck was complaining about.
The $daos_logs could be "foo.log 'bar baz.log' abc.log" etc.

The existing code only works if we don't have spaces in the log file names.

Reply:
The answer was to move the lbzip2 into the find command to use xargs.
Pushed an update.

Reply:
This is fixed in the current patches.